### DISCUSSION TITLE
Upgrading "dependencies" also requires updating package.json, but renovate only generates PR for lockfile updates

### DISCUSSION THREAD

**Discussion by JellyBrick** [renovatebot/renovate#23366] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

- It works fine in [other repositories](https://github.com/KiwiTalk/KiwiTalk) (that I own) with the same renovate settings, but not in [this one](https://github.com/organization/alspotron).

### Describe the problem

- Only updated PRs for the lock-file are generated.
- e.g. https://github.com/organization/alspotron/pull/77
- Vite is a dependency defined in package.json, but renovate only generates lock-file updates.
- repository link: https://github.com/organization/alspotron

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
This is correct behavior as you have a range in your package.json which doesn't need changing. 

Read the docs on "rangeStrategy" for more information and options
